.github/workflows/test.yml

```diff
@@ -76,7 +76,7 @@ jobs:
         name: Setup Cargo Toolchain 🛎️
         with:
           components: rustfmt, clippy, rust-src
-          toolchain: nightly
+          toolchain: stable
```

Comment:
Running sanitize test should use nightly toolchain

Reply:
got it